packages/calcite-components/src/components/dropdown/dropdown.tsx

```diff
@@ -237,7 +237,11 @@ export class Dropdown
   render(): VNode {
     const { open, guid } = this;
     return (
-      <Host>
+      <Host
+        {...{
+          "aria-label": this.label,
```

Comment:
can this be moved onto the `calcite-trigger-container` element so that its within the shadowDOM?

Reply:
aria- properties don't work when used on shadowDOM elements

Reply:
Hey @alisonailea, Is it just not reading correctly? We have many other aria attributes inside shadowDOM of components so I think this should be doable.

The reason I think it would be better in the shadowDOM is so that an end user doesn't have the ability to override it or remove it accidentally. We try not to expose these kind of internals outside of the component.